src/index.ts

```diff
@@ -59,4 +59,16 @@ export class AgnosticCMSHarmonizerClient {
       throw new Error(`Error obtaining entry:\n${error}`);
     }
   }
+
+  protected async getEntriesHarmonized<T = Record<string, unknown>>(
```

Comment:
Should we document it?

Reply:
Absolutely, I'm doing it in a followup branch already 🚀